packages/sources/css/config/postcss/README.md

```diff
@@ -1,3 +1,11 @@
 # `@vtmn/css-postcss-config` (private)
 
 > Decathlon Design System - Vitamin CSS PostCSS configuration
+
+## I have an issue, what should I do?
+
+Please file the issue [here](https://github.com/Decathlon/vitamin-web/issues/new).
+
+## Contributing
+
+See [the contributing docs](https://github.com/Decathlon/vitamin-web/blob/main/CONTRIBUTING.md).
```

Comment:
Same in these types of files, I guess it is no longer useful to re-specify the license?

Reply:
We need it because each packages is deployed indepentely :/